cfgen/cfgen

```diff
@@ -803,8 +803,15 @@ class ConfProcess(ToDhall):
                               services=[])
         m0conf[parent].processes.append(proc_id)
 
+        if proc_t is ProcT.m0_client_other:
+            params = ["origin:in-volatile"]
+        if proc_t is ProcT.m0_server:
+            params = ["origin:in-persistent"]
+
         for stype in service_types(proc_t, proc_desc):
-            svc_id = ConfService.build(m0conf, proc_id, stype, ep)
+            sparams = params if stype is SvcT.M0_CST_DTM0 else []
```

Comment:
`params` may be None at this point. Is it expected?

Reply:
Initialized as [].